tests/test_math/test_compactFock.py

```diff
@@ -9,7 +9,11 @@
 from mrmustard.lab import DM, Ggate, SqueezedVacuum, Vacuum
 from mrmustard.lab.transformations.attenuator import Attenuator
 from mrmustard.physics import gaussian
-from mrmustard.training import Optimizer
+
+try:
+    from mrmustard.training import Optimizer
+except ImportError:
+    Optimizer = None
```

Comment:
I might have thought that `test_compactFock_1leftover_gradients` should have the `@pytest.mark.requires_backend("jax")` decorator, just like `test_compactFock_diagonal_gradients`. But it can't yet? What ends up getting tested?